internal/provisioner/helm/helm.go

```diff
@@ -21,7 +21,7 @@ const (
 	ProvisionerID = "core-rukpak-io-helm"
 )
 
-func HandleBundle(ctx context.Context, fsys fs.FS, bundle *rukpakv1alpha1.Bundle) (fs.FS, error) {
+func HandleBundle(_ context.Context, fsys fs.FS, _ *rukpakv1alpha1.Bundle) (fs.FS, error) {
```

Comment:
Here and in other places with `_` args - we don't use args in functions, but we still need args to statisfy interfaces.